checks/raw/pinned_dependencies.go

```diff
@@ -405,10 +417,21 @@ var validateGitHubWorkflowIsFreeOfInsecureDownloads fileparser.DoWhileTrueOnFile
 
 // Check pinning of github actions in workflows.
 func collectGitHubActionsWorkflowPinning(c *checker.CheckRequest, r *checker.PinningDependenciesData) error {
-	return fileparser.OnMatchingFileContentDo(c.RepoClient, fileparser.PathMatcher{
+	err := fileparser.OnMatchingFileContentDo(c.RepoClient, fileparser.PathMatcher{
 		Pattern:       ".github/workflows/*",
 		CaseSensitive: true,
 	}, validateGitHubActionWorkflow, r)
+	if err != nil {
+		return err
+	}
+
+	//nolint:errcheck
+	remediationMetadata, _ := remediation.New(c)
+	for i := range r.Dependencies {
+		rr := r.Dependencies[i]
+		rr.Remediation = generateRemediation(remediationMetadata, &rr)
```

Comment:
same comments as above

Reply:
Thanks for catching that. Done in https://github.com/ossf/scorecard/pull/3632/commits/227bcc14c019076e3d94a67f30b5e9fada781b7c